python/ray/data/read_api.py

```diff
@@ -562,6 +562,15 @@ def read_csv(
         >>> ray.data.read_csv( # doctest: +SKIP
         ...     ["s3://bucket/path1", "s3://bucket/path2"])
 
+        >>> # Use different delimiter. The partition_filter=None is needed here
```

Comment:
nit: For consistency with other examples

```suggestion
        >>> # Read files that use a different delimiter. The partition_filter=None is needed here
```